docs/src/test-global-setup-teardown-js.md

```diff
@@ -135,13 +146,7 @@ You can use the `globalSetup` option in the [configuration file](./test-configur
 Similarly, use `globalTeardown` to run something once after all the tests. Alternatively, let `globalSetup` return a function that will be used as a global teardown. You can pass data such as port number, authentication tokens, etc. from your global setup to your tests using environment variables.
 
 :::note
-Beware of `globalSetup` and `globalTeardown` caveats:
-
-- These methods will not produce traces or artifacts unless explictly enabled, as described in [Capturing trace of failures during global setup](#capturing-trace-of-failures-during-global-setup).
-- Options such as `headless` or `testIdAttribute` specified in the config file are not applied.
-- An uncaught exception thrown in `globalSetup` will prevent Playwright from running tests, and no test results will appear in reporters.
-
-Consider using [project dependencies](#option-1-project-dependencies) to produce traces, artifacts, respect config options and get test results in reporters even in case of a setup failure.
+Beware that `globalSetup` and `globalTeardown` lack some features — see the [intro](#introduction) section for a detailed comparison. For example, they do not respect config options like `headless` or `testIdAttribute`, won't produce traces or artifacts unless explicitly configured, and uncaught exceptions will prevent tests from running with no results reported. Consider using [project dependencies](#option-1-project-dependencies) instead to get full feature support.
```

Comment:
I'd remove the details from here to avoid repetition: "beware they lack some features, see intro for details, consider using project dependencies".

Reply:
Done.